packages/reports/addon/components/filter-builders/metric.ts

```diff
@@ -29,12 +29,12 @@ export default class MetricFilterBuilderComponent extends BaseFilterBuilderCompo
         component: 'filter-values/value-input',
       },
       {
-        operator: 'eq' as const,
```

Comment:
This doesn't fix the `number` dimension type since it uses the `dimension` filter builder. You should start by adding a mock `number` type dimension add it as a filter, and make it show up with the same filters listed here